xAPI-About.md

```diff
@@ -540,23 +546,20 @@ move those changes into the current specification and revision to 0.9.
 
 "Core" Verbs and Activity types were removed from the specification. References 
 to these verbs in results, context, interactions, and Activity Definitions were 
-also  removed. It was recommended that implementers prefer community defined 
+also removed. It was recommended that implementers prefer community defined 
 verbs to creating their own verbs.
 - Verbs, Activity types, and extension keys are now URIs
-- Restructured and added language around some of the other implementation 
-details and scope.
-- Changed from using a person-centric view of Agents to a persona-centric 
-view.
+- Restructured and added language around some of the other implementation details and scope.
+- Changed from using a person-centric view of Agents to a persona-centric view.
 - Friend of a Friend (FOAF) Agent merging requirement was removed.
-- Agent Objects now have exactly 1 uniquely identifying property, instead 
-of at least one.
+- Agent Objects now have exactly 1 uniquely identifying property, instead of at least one.
 
 ###### 0.95 to 1.0.0 (April 26, 2013) 
 Various refinements and clarifications including:
-- Adding attachments
+- Adding Attachments
 - Activity metadata is now stored as JSON rather than XML
 - Changes to voiding Statements
-- Clarification and naming of the Document APIs
+- Clarification and naming of the Document APIs (now Document Resources)
```

Comment:
Can we change "now" to "later renamed to" to avoid suggesting that they changed to Document Resources  in 1.0.0?